packages/docs/docs/compliance/clia-cap.md

```diff
@@ -40,6 +40,7 @@ Review and refer to the following items in preparation for your inspection.
   - [ ] Specimen related queries and reports
     - [ ] All [Specimens](https://app.medplum.com/Specimen)
     - [ ] [Specimen Quality](https://app.medplum.com/Specimen?_count=20&_fields=id,_lastUpdated,status&_offset=0&_sort=-_lastUpdated&status=unsatisfactory)
+    - [ ] [Specimen.condition](https://app.medplum.com/Specimen?_count=20&_fields=id,_lastUpdated,condition&_offset=0&_sort=-_lastUpdated) should include codes from the [`SpecimenRejectionReason` code system](https://terminology.hl7.org/CodeSystem-v2-0490.html) that will be the basis for rejecting.
```

Comment:
This is a legacy HL7 2.*, which is the current guidance. 

USCDI V4 will bring some clarity to the ontology required here. Right now there's a debate whether to require SNOMED codes exclusively, or to allow a combination of both SNOMED and HL7 2.* codes.

For some user implementations, we define both SNOMED and HL7 2.* codes for these concepts (see [this doc](https://docs.google.com/document/d/1zPDnHO3pCqRF5rBVpVgSfbVeoNcXolhGOGcAPGgY_qI/edit?usp=sharing))

[reference](https://www.healthit.gov/isa/taxonomy/term/7691/draft-uscdi-v4)